Add stylelint (#37)
googleinterns/gpay-group-buy: web/README.md

### `yarn lint`

> Lints and checks code for formatting problems according to [Google Typescript Style](https://github.com/google/gts). Lints CSS as well.

Reviewer: should we add a link for the CSS linter also?

Author: @karenfrilya97 Hmmmm I'm not sure, gts comes with its set of rules (aka opinionated in some ways), but for the css linter i'm just using the standard lint rules so I'm not so sure hmm.
@ifhuang @shengdac @erencie  Any inputs on this? 😛

Reviewer 2: Actually why do we need this stylelint? gts doesn't cover it?

Author: @ifhuang Nope, gts is not catered towards frontend dev specifically so CSS linter is not included! gts uses eslint and prettier behind the hood, those tools lint and format code, but not styles!

Reviewer 2: ic. adding a link for the CSS linter SGTM

> Lints and checks code for formatting problems according to [Google Typescript Style](https://github.com/google/gts). Lints CSS as well.

Author: ```suggestion
Lints and checks TypeScript code for formatting problems according to [Google Typescript Style](https://github.com/google/gts), and lints CSS code inside styled components using [stylelint](https://stylelint.io/) with the [standard config rules](https://github.com/stylelint/stylelint-config-standard).
```

#### `yarn lint:code`

> Lints code according to [Google Typescript Style](https://github.com/google/gts) only.

Author: ```suggestion
Lints and checks TypeScript code for formatting problems according to [Google Typescript Style](https://github.com/google/gts) only.
```

#### `yarn lint:css`

> Lints CSS in styled components.

Author: ```suggestion
Lints CSS in styled components using [stylelint](https://stylelint.io/) with the [standard config rules](https://github.com/stylelint/stylelint-config-standard).
```

> Lints CSS. Run ``yarn lint:css --fix`  to fix lint errors automatically (if possible).

Reviewer 2: extra `?

Author: Good catch! My .md files are bugging so bad on my vscode I didn't even realise this 😭 😢

Author: Fixed it!
